src/components/utils/Author.js

```diff
@@ -0,0 +1,10 @@
+const Author = () => (
+  <div className="form-control author">
+    <select id="author" className="form-control-author bg-white " name="author">
+      <option value="wael-elsafty">Wael Elsafty</option>
+      <option value="seif-ali">Seif Ali</option>
+    </select>
+  </div>
+);
```

Comment:
- [ ] Dear @waelelsafty07, Great job on using a `util` for rendering such components, But please notice the `Author` field is an `input` element with a type of `text`. Please change the `select` type to the `input` type. we need to be able to write the author's information.